### DISCUSSION TITLE
使用gomobile 命令编译Xray-core 生成iOS移动端的库文件的时候报错

### DISCUSSION THREAD

**Discussion by 594810899** [XTLS/Xray-core#1073] [Q&A]
使用gomobile 命令编译Xray-core 生成iOS移动端的库文件   
cd $GOPATH/src/golang.org/x/mobile/example/bind
gomobile bind -target=ios golang.org/x/mobile/example/bind/Xray-core 的时候 报错返回no exported names in the package "golang.org/x/mobile/example/bind/Xray-core"  这是哪里出了问题  请求哪位知道的 指点一下，非常感谢！

**Comment by linchuIOS** [comment]
any updates?

**Reply by alikashan** [reply]
I am also getting the same error
